ansible/roles/ocp4-workload-camelk-lab/templates/installed-community-openshift-operators.yml

```diff
@@ -1,13 +1,14 @@
+---
```

Comment:
This is odd... `.yml` file using all json syntax? Technically valid, but should be just using YAML syntax.

Reply:
I'm not sure if the build is failing bc of the json to yaml conversion or something else..